hooks/post-checkout

```diff
@@ -1,112 +1,40 @@
 #!/bin/bash
 
-set -Eeuo pipefail
+# Load common tools
```

Comment:
```suggestion
set -Eeuo pipefail

 # Load common tools 
```

Reply:
We get the `set -Eeuo pipefail` from `common.sh`